lib/task.rb

```diff
@@ -1,14 +1,11 @@
-class Task
-    attr_reader :title, :description, :is_complete
+require "sequel"
+require_relative "Database"
 
-    def initialize(title, description)
-        @title = title
-        @description = description
-        @is_complete = false
+class Task < Sequel::Model(:tasks)
+    plugin :validation_helpers
+    def validate
```

Comment:
Now that you have validation, you'll need to catch validation errors & uniqueness errors.